scripts/preconversion_assessment_script.py

```diff
@@ -197,27 +258,115 @@ def cleanup(required_files=None):
         )
 
 
-def verify_required_files_are_present(required_files=None):
-    if required_files is None:
-        required_files = REQUIRED_FILES
-
+def verify_required_files_are_present(required_files):
+    """Verify if the required files are already present on the system."""
+    print("Checking if required files are present on the system.")
```

Comment:
I moved it the other way around... All prints are in the functions itself rather than the main one. 

I think that having it like that makes it more clean and encapsulates the message to be inside each function.

Reply:
okay, looks great in the main now!